### DISCUSSION TITLE
Ensemble-stat METplus use case without product generation

### DISCUSSION THREAD

**Discussion by michelleharrold** [dtcenter/METplus#1543] [Use Cases] [answered]
Good Morning,

In the UFS SRW App regional workflow, we currently use the ensemble-stat tool for both ensemble product generation and calculating ensemble stats. We are transitioning to use both gen-ens-prod and ensemble-stat. There is a great example of a gen-ens-prod METplus use case, but I was wondering if you had a use case for ensemble-stat without the product generation capabilities (I want to make sure I remove all unnecessary entries in the ensemble-stat METplus configuration file - I know the product generation capability is still supported in ensemble-stat, but I want to make a clean break).

Thanks!
Michelle Harrold

**Comment by j-opatz** [comment, marked as answer]
Hi Michelle,

And thank you for your request. After a discussion with the METplus engineering lead, it seems like this is a good idea to incorporate into METplus and provide an example of the new functionality of EnsembleStat.

As a result, we've created a new Github [issue](https://github.com/dtcenter/METplus/issues/1546) that you can track that will result in the creation of a new use case.

This issue may be moved to a different beta depending on resource management, but we will provide updates to the issue as time continues.

**Comment by michelleharrold** [comment]
Thank you for the reply @j-opatz! Sounds like a plan -- if you need a beta tester, let me know!
